### ISSUE TITLE
Unable to convert OpenVino Facenet-20180408-102900 model to tflite

### ISSUE THREAD

**Issue by maurobento** [PINTO0309/openvino2tensorflow#56]
Hi, first of all thanks for your work, it's amazing!

Im using the docker image in Ubuntu 20.04 to convert the Facenet-20180408-102900 model in OpenVino format to tflite but i'm having some issues that i cannot resolve.

I added a json file to correct the errors due to reshape and transpose operations
The json file looks like this:
```
{
    "format_version": 1,
    "layers": [
        {
            "layer_id": "728",
            "replace_mode": "direct",
            "values": [
                1,
                1,
                512,
                512
            ]
        }
    ]
}

```
But then, when i try to convert the model this massege comes up:
```
The NormalizeL2 layer is not yet implemented.
```
It's obvious that the NormalizeL2 operation is not implemented yet, but maybe you have a workarround to this problem.

Where can i send you the .bin and .xml files so you can try them?

Thanks

Mauro

**Comment by PINTO0309** [comment]
Please provide a model if you can. I can't reproduce the problem.

**Comment by maurobento** [comment]
Hi, i sent you an email to this addres "rmsdh122@yahoo.co.jp" with a link to download the network because is to heavy to attach here. Let mi know if there is any problem downloading the files.

Thanks

**Comment by PINTO0309** [comment]
If you're using this model, which is MIT licensed, then I see no point in you hiding the URL. Anyway, I will check the conversion.
https://docs.openvinotoolkit.org/latest/omz_models_model_facenet_20180408_102900.html

**Comment by PINTO0309** [comment]
Fixes: cefbef972efdc47861ab4c1cbd936bca812d49f1
https://github.com/PINTO0309/openvino2tensorflow/releases/tag/v1.16.0
